main.cpp

```diff
@@ -200,166 +217,152 @@ struct iPhone FIXME: type names need to begin with a capital letter
-    1) Play tape
-    2) Show timecode
-    3) Change speed in IPS
- */
+void MyRefrigerator::changeFridgeTemperature(float newTemperature)
+{
+    tempOfFridge = newTemperature;
+}
+
+void MyRefrigerator::changeFreezeTemperature(float newTemperature)
+{
+    tempOfFreezer = newTemperature;
+}
+
+int MyRefrigerator::numberOfItemsInFridge()
+{
+    return numberOfEggs + numberOfVegetables;
+}
 
+// --------------------------------------------------------------------------
 struct TapeMachine
 {
-//     1) Number of tracks (int)
     int numberOfTracks = 24;
-//     2) Current speed in IPS (float)
-    float speedInIPS = 30.0f;
-//     3) Current timecode (double)
-    double currentTime = 0.00;
-//     4) Current play status (std::string)
+    float currentSpeedInIPS = 30.0f;
+    double currentTimecode = 0.00;
     std::string playStatus = "stopped";
-//     5) Model of Machine (std::string)
     std::string model = "Studer";
-//     1) Play tape
     void play(float speedInIPS, double fromTime);
-//     2) Show timecode
-    double showTimeCode(double currentTime); // returns the current timecode
-//     3) Change speed in IPS
+    double showTimeCode(double currentTime);
     void changeSpeedInIPS(float newSpeed);
 };
 
-/*
-Thing 5) Display
-    1) Number of pixels (int)
-    2) Brightness level (float)
-    3) Width (float)
-    4) Height (float)
-    5) Energy consumed (float)
-    1) Show visuals
-    2) Change brightness
-    3) Consume energy
- */
+void TapeMachine::play(float speedInIPS, double fromTime)
+{
+    std::cout << "Playing tape from " << fromTime << " at " << speedInIPS << " inches per second."<< std::endl;
+}
+
+double TapeMachine::showTimeCode(double currentTime)
+{
+    currentTimecode = currentTime;
+    return currentTimecode;
+}
 
+void TapeMachine::changeSpeedInIPS(float newSpeed)
+{
+    currentSpeedInIPS = newSpeed;
+}
+
+// --------------------------------------------------------------------------
 struct Display
 {
-//     1) Number of pixels (int)
     int numberOfPixels = 1080;
-//     2) Brightness level (float)
     float brightnessLevel = 75.0f;
-//     3) Width (float)
     float width = 3.5f;
-//     4) Height (float)
     float height = 3.0f;
-//     5) Energy consumed (float)
     float energyConsumed = 50.0f;
-//     1) Show visuals
     void showVisuals(int pixelsToChange);
-//     2) Change brightness
     void changeBrightness(float newBrightnessLevel);
-//     3) Consume energy
     void consumeEnergy (float amountOfEnergyToConsume = 0);
 };
 
-/*
-Thing 6) Controls
-    1) Number of Start Buttons (int)
-    2) Number of Select Buttons (int)
-    3) Number of D pads (int)
-    4) Number of other buttons (int)
-    5) Energy consumed (float)
-    1) Take input from user
-    2) Pause game
-    3) Start Game
- */
+void Display::showVisuals(int pixelsToChange)
+{
+    for(int i = 0; i < pixelsToChange; i++)
+        {
+            std::cout << "Changed Pixel!" << std::endl;
+        }
+}
```

Comment:
loops have not been covered yet.
Please remove this and write something else instead.
- [ ] revise